test/qa/test_login.py

```diff
@@ -28,6 +31,98 @@ def teardown_function(function):  # noqa: ARG001
     logging.log()
 
 
+def test_analytics_consent_is_displayed_on_login():
+    cli = pexpect.spawn("nordvpn", args=["login"], encoding='utf-8', timeout=10)
```

Comment:
Care to extract common parts of the tests into a fixture? Possibly spawning the application, some of the `cli.expect` calls and possibly buffers could be unified.

Shouldn't be a lot of changes I guess (if it happens to be other way round, please disregard my comment then!).

Reply:
Done